### PR TITLE
NO-ISSUE - Make _EnvVariablesUtils singleton.

### PR DESCRIPTION
**Pull request by eliorerz**
Rename _EnvVariablesUtils to _EnvVariablesDefaults

### CODE REVIEW

File: discovery-infra/test_infra/helper_classes/config/controller_config.py [hunk 1]
```diff
@@ -5,11 +5,8 @@
 from dataclasses import dataclass
 
 from test_infra import consts
-from test_infra.utils.global_variables import GlobalVariables
 from .base_config import _BaseConfig
 
-global_variables = GlobalVariables()
```
Comment (YuviGold) [inline discovery-infra/test_infra/helper_classes/config/controller_config.py:11]: Where is it being assigned instead?

Comment (eliorerz) [reply discovery-infra/test_infra/helper_classes/config/controller_config.py:11]: Where it was initiated from the first place. In `global_configs.py`

**Review by YuviGold — APPROVED** [review]

**Pull request merged**